test/wasmBenchmarker/ctests/rsa.c

```diff
@@ -228,22 +228,134 @@ int runtime() {
                             "te eum quod blandit. Vim ad legimus intellegebat "
                             "disputationi, an ridens nonumes deterruisset sed. Eos "
                             "dicunt assentior eu, mel ne case postulant, "
-                            "quando feugiat voluptaria eam ne.\n";
+                            "quando feugiat voluptaria eam ne.\n"
```

Comment:
Is this a duplication, or the continuation of lorem ipsum?

Reply:
I've generated Lorem Ipsum, and copied it from a part. I think there is some repetitive parts in lipsum, for example, the `Lorem ipsum dolor sit amet`, but I've searched `Integer`, and at the beginning of text, there is no word `integer`